kasa/tapo/tapobulb.py

```diff
@@ -166,13 +166,7 @@ async def set_hsv(
         if value is not None:
             request_payload["brightness"] = value
 
-        return await self.protocol.query(
-            {
-                "set_device_info": {
-                    **request_payload
-                }
-            }
-        )
+        return await self.protocol.query({"set_device_info": {**request_payload}})
```

Comment:
I'm wondering if we should convert `query` to use a similar format than what's used for the "legacy" protocol with its `_query_helper()`? Basically, having a method with a signature of `_something(command, params)` to avoid dict construction on the callsites.

Reply:
Agreed, I'm planning to do something like this shortly in a separate PR when I enable multi requests.